build-support/githooks/prepare-commit-msg

```diff
@@ -16,22 +16,22 @@ MERGE_BASE="$(git_merge_base)"
 # new build.
 CHANGED_FILES="$(git diff --name-only "${MERGE_BASE}")"
 
-NUM_NON_MD_FILES=$(echo "${CHANGED_FILES})" | grep -c -v ".\md$")
-
 # Ensure that this stays in sync with `build-support/bin/rust/calculate_engine_hash.sh`.
 NUM_RUST_FILES=$(echo "${CHANGED_FILES})" | grep -c -E \
   -e "^src/rust/engine" \
   -e "^rust-toolchain" \
   -e "^build-support/bin/rust" \
-  -e "^build-support/bin/generate_travis_yml.py")
+  -e "^build-support/bin/generate_travis_yml.py" \
+  -e "^build-support/bin/generate_github_workflows.py")
```

Comment:
I want to replace this with GitHub Action's first-class support for this, but in the meantime..